### ISSUE TITLE
Docker-compose.yml does not work

### ISSUE THREAD

**Issue by SylvainDumas** [microcks/microcks#1289]
### Describe the bug

Microcks SSO fails because he can not use string `start-dev --import-realm` as shell arguments for his script.

### Expected behavior

Keycloak works and does not exit

### Actual behavior

Errors in logs
`
microcks-sso              | Option: 'start-dev --import-realm' is not expected to contain whitespace, please remove any unnecessary quoting/escaping
microcks-sso              | Did you mean: kc.sh start-dev or kc.sh import or kc.sh start?
microcks-sso              | Try 'kc.sh --help' for more information on the available options.
microcks-sso exited with code 2
`

### How to Reproduce?

Go to folder `install/docker-compose`

Run
```sh
docker-compose -f docker-compose.yml up
```

### Install method (`docker-compose`, `helm chart`, `operator`, `docker-desktop extension`,...)

docker-compose

### Additional information

Prefer using syntax as an array
```yaml
    command: ["start-dev", "--import-realm"]
```

Instead of
```yaml
    command:
      - start-dev --import-realm
```

See [ref](https://docs.docker.com/reference/compose-file/services/#command)

This fix for docker-compose.yml, but may be necessary in others, to check

**Comment by lbroudoux** [comment]
OMG 🥹 You're right, I can't believe we screwed this one. Certainly during our last Keycloak update... Gonna fix it asap.

**Comment by lbroudoux** [comment]
It has been fixed on `1.10.x` but will be also reported on `master`.
